lib/denormalizer.js

```diff
@@ -714,9 +717,12 @@ _.extend(Denormalizer.prototype, {
    *                            `function(err, evt){}` evt is of type Object.
    */
   getLastEvent: function (callback) {
-    this.revisionGuardStore.getLastEvent(callback);
+    this.revisionGuardStore.getLastEvent(this.options.revisionGuard.prefix, callback);
   },
 
+  getLastEventOfEachAggregate: function (callback = (err, aggregateHandleFns) => {}) {
+    this.revisionGuardStore.getValueOfEachKey(this.options.revisionGuard.prefix, callback);
```

Comment:
idk, getValueOfEachKey sounds very specific to the redis implementation...
What about mongodb, etc...?

Reply:
how would you like getValueOfEachId?

Reply:
@nanov Any better idea?

Reply:
I will now add the getValueOfEachId version and send you update the PR :)